src/app/home/countdown/countdown.component.html

```diff
@@ -0,0 +1,81 @@
+<div *ngIf="!isFinishDate" class="hide-on-small-only">
+  <div class="row container">
+    <div class="col s12 center-align">
+      <h2 id="countdown-title" class="medium-text"> {{'countdown.title' | translate}} </h2>
```

Comment:
```suggestion
      <h2 id="countdown-title" class="medium-text">{{ 'countdown.title' | translate }}</h2>
```

Reply:
In fact, try to put the title like it is in other sections. You car review, sponsors or partners section and put the title like in those sections in order to have a standard.